apps/info/admin.py

```diff
@@ -191,6 +193,17 @@ class PlaceAdmin(admin.ModelAdmin):
 class PressReleaseAdmin(admin.ModelAdmin):
     list_display = ("festival",)
 
+    def get_form(self, request, obj=None, **kwargs):
+        """Set free festivals plus current festivals."""
+        form = super().get_form(request, obj, **kwargs)
+        change_permission = get_user_change_perms_for_status(request, obj)
+        if change_permission:
+            select = None
+            if obj:
+                select = obj.festival_id
+            form.base_fields["festival"].queryset = get_vacant_and_current_festival(select)
```

Comment:
Не вижу особого смысла делать функцию, да ещё в другом файле. Мне кажется, что так тоже заработает
```suggestion
            form.base_fields["festival"].queryset = Festival.objects.filter(Q(press_releases__festival__isnull=True) | Q(id=None if not obj else obj.festival_id))
```
можно только расчёт id вынести в переменную, чтобы упростить восприятие выражение